php1/mysql/class-mail.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+session_start();
```

Comment:
According to sir, you shoud not start the `session` in class file so please check it

Reply:
But I had to access a local variable which is within a function without creating an object of the class that's the reason I used session still I will try to look for a way to rectify this.

Reply:
You can access the variable in another way like using return type or declared a varible in class level and store the value after that access it with the help of object etc..